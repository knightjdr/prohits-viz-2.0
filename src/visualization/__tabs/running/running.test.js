import React from 'react';
import { shallow } from 'enzyme';

import Running from './running';

describe('Visualization running component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Running />);
    expect(wrapper).toMatchSnapshot();
  });
});
