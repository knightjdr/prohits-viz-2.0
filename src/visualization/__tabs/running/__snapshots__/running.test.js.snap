// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Visualization running component should match snapshot 1`] = `
<div
  className="visualization__running"
>
  <Spin
    prefixCls="ant-spin"
    size="large"
    spinning={true}
    wrapperClassName=""
  />
</div>
`;
